From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Karthik Prabhu Vinod <karthik.prabhu.vinod@intel.com>
Date: Fri, 8 Jan 2021 13:43:54 -0800
Subject: [PATCH] Add missing requirements to pc file

Signed-off-by: Karthik Prabhu Vinod <karthik.prabhu.vinod@intel.com>
Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 meta/guile-3.0.pc.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/meta/guile-3.0.pc.in b/meta/guile-3.0.pc.in
index c6d12b5..3a11e49 100644
--- a/meta/guile-3.0.pc.in
+++ b/meta/guile-3.0.pc.in
@@ -22,6 +22,8 @@ guile=${bindir}/@guile@
 Name: GNU Guile
 Description: GNU's Ubiquitous Intelligent Language for Extension
 Version: @GUILE_VERSION@
+Requires: bdw-gc
+Requires.private: gmp libffi
 Libs: -L${libdir} -lguile-@GUILE_EFFECTIVE_VERSION@ @BDW_GC_LIBS@
 Libs.private: @LIB_CLOCK_GETTIME@ @LIBGMP@ @LIBLTDL@ @LIBFFI_LIBS@	\
   @LIBUNISTRING@ @GUILE_LIBS@ @LIBICONV@ @LIBINTL@ @LIBSOCKET@		\
