From 4c5ab62ab2ea423375144bbf476646e493625093 Mon Sep 17 00:00:00 2001
From: Karthik Prabhu Vinod <karthik.prabhu.vinod@intel.com>
Date: Fri, 8 Jan 2021 13:43:54 -0800
Subject: [PATCH] Add Require for guile on bdw-gc

Signed-off-by: Karthik Prabhu Vinod <karthik.prabhu.vinod@intel.com>
---
 meta/guile-2.2.pc.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/meta/guile-2.2.pc.in b/meta/guile-2.2.pc.in
index c6d12b5..2e6e800 100644
--- a/meta/guile-2.2.pc.in
+++ b/meta/guile-2.2.pc.in
@@ -22,6 +22,7 @@ guile=${bindir}/@guile@
 Name: GNU Guile
 Description: GNU's Ubiquitous Intelligent Language for Extension
 Version: @GUILE_VERSION@
+Requires: bdw-gc
 Libs: -L${libdir} -lguile-@GUILE_EFFECTIVE_VERSION@ @BDW_GC_LIBS@
 Libs.private: @LIB_CLOCK_GETTIME@ @LIBGMP@ @LIBLTDL@ @LIBFFI_LIBS@	\
   @LIBUNISTRING@ @GUILE_LIBS@ @LIBICONV@ @LIBINTL@ @LIBSOCKET@		\
-- 
2.30.0

